presto-docs/src/main/sphinx/release/release-0.197.rst

```diff
@@ -0,0 +1,65 @@
+=============
+Release 0.197
+=============
+
+General Changes
+---------------
+
+* Fix query scheduling hang when the ``concurrent_lifespans_per_task`` session property is set.
+* Fix query failure when query contains a ``TIMESTAMP`` literal corresponding to a local time
+  that does not occur in the time zone of the Presto JVM. For example, if Presto was run in a
+  CET zone (e.g., ``Europe/Brussels``) and the client session was in UTC, an expression
+  such as ``TIMESTAMP '2017-03-26 02:10:00'`` would cause a failure.
+* Extend predicate inference and pushdown for queries using a ``<symbol> IN <subquery>`` predicate.
+* Support predicate pushdown for the ``<column> IN <values list>`` predicate
+  where values in the ``values list`` require casting to match the type of ``column``.
+* Optimize :func:`min` and :func:`max` to avoid unnecessary object creation in order to reduce GC overhead.
+* Optimize the :func:`ST_XMin`, :func:`ST_XMax`, :func:`ST_YMin` and :func:`ST_YMax` functions.
+* Add ``DATE`` variant for :func:`sequence` function.
+* Add :func:`ST_IsSimple` geospatial function.
+* Add support for broadcast spatial joins.
+
+Resource Groups Changes
+-----------------------
+
+* Change configuration check for weights in resource group policy to validate that
+  either all of the subgroups or none of the subgroups have a scheduling weight configured.
+* Add support for named variables in source and user regular expressions that can be
+  used to parameterize resource group IDs.
```

Comment:
What is `resource group ID` (there is no ID property in the resource groups docs, so the users don't know it)?  I guess what we want to say is "resource group names".